examples/dgfip_c/backend_tests/test_harness.c

```diff
@@ -86,7 +89,10 @@ int main(int argc, char *argv[])
                         // Here we move to controlling the outputs, so we
                         // have to run the computation!
                         m_input_from_array(input_for_m, input_array_for_m);
+                        errors = malloc(sizeof(Errors));
```

Comment:
I still can't see where this `Errors` live, it does not appear to be generated in `bir_to_dgfip_c.ml`.

Reply:
As mentioned IRL, Errors is typedef of an m_error table of the size of the number of errors possible during a calculation.